index.html

```diff
@@ -128,15 +138,62 @@ <h2>ts-root</h2>
 				</script>
 			</article>
 
+			<article>
+				<h2>ts-note</h2>
+				<ts-note type="neutral" icon="info"><i>this</i> is <b>easy</b> to <a href="#use">use</a>.</ts-note>
+				<ts-note type="neutral" rtl icon="info">
+					متن پیغام خطا برای کاربر
+				</ts-note>
+				<ts-note icon="info" type="ada">
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ad aliquid asperiores distinctio eligendi eum ipsum
+					iure libero maxime modi perspiciatis quam quis repellendus repudiandae sequi sunt ut, veritatis voluptatem
+					voluptatibus!
+				</ts-note>
+				<ts-note type="ada" rtl icon="info">
+					متن پیغام خطا برای کاربر
+				</ts-note>
+				<ts-note icon="info" closeable id="closeableNote1">
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ad aliquid asperiores distinctio eligendi eum ipsum
+					iure libero maxime modi perspiciatis quam quis repellendus repudiandae sequi sunt ut, veritatis voluptatem
+					voluptatibus! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Delectus enim ipsum officia omnis quo
+					vero voluptatibus. Eius eum, illum ipsum iste laboriosam magnam natus neque nisi obcaecati officiis quo,
+					repellat. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consequatur deleniti ducimus error fugiat
+					ipsam nostrum possimus quo totam velit. At atque consequuntur corporis cum incidunt quidem quis quisquam
+					repellat voluptatibus! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsam nostrum, perspiciatis!
+					Ab ad aliquam blanditiis debitis dolor earum error, exercitationem illo maiores, mollitia optio perferendis
+					rem repellendus, totam voluptates voluptatibus. Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+					Consequatur in obcaecati provident quo repudiandae sapiente sint ut? Cupiditate deserunt nemo nobis recusandae
+					vel. Beatae esse laboriosam officia sequi voluptas voluptatibus.
+				</ts-note>
+
+				<ts-note id="closeableNote2" rtl closeable icon="info">
+					متن پیغام خطا برای کاربر
+				</ts-note>
+				<script>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The length of this JS script (12) exceeds the maximum set to 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3ALongJavaScriptCheck)